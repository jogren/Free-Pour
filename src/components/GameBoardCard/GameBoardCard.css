.Game-card_div {
  height: 70px;
  margin-bottom: 10px;
  border-radius: 5px;
  width: 80%;
  display: flex;
  justify-content: center;
  align-items: center;
}

.incorrect-guess {
  border: 2px solid black;
}

.correct-guess {
  background-color: black;
  color: #FFF;
}

.Game-card_div p {
  font-size: 24px;
  font-weight: 700;
}

.GameBoard_main {
  display: flex;
  flex-direction: column;
  align-items: center;
}

.GameBoard_main h3 {
  margin: 25px;
  font-size: 24px;
}

.GameBoardCard_section {
  width: 100%;
  display: flex;
  justify-content: center;
}

.Board_section-ingredients button {
  border-radius: 0px 5px 5px 0px;
  height: 99%;
  padding-top: 2px;
}

.Board_section-ingredients button:hover {
  background-color: black;
  color: #FFF;
}

.Board_section-ingredients {
  height: 30px;
}

.div-round-finished {
    display: flex;
    flex-direction: column;
    align-items: center;
}

.div-round-finished h4 {
  margin: 30px 0px 10px 0px;
  font-size: 32px
}

.div-round-finished button {
  width: 180px;
  height: 35px;
  font-size: 16px;
}

.div-round-finished button:hover {
  background-color: black;
  color: #FFF;
}


