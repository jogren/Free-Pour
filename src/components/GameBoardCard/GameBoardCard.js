import React from 'react';
import './GameBoardCard.css';

const GameBoardCard = ({ ingredient, index }) => {

  return (
    <section>
      {!ingredient.guessed && <div className="Game-card_div incorrect-guess">
        <p>{index + 1}</p>
      </div>}
      {ingredient.guessed && <div className="Game-card_div correct-guess">
        <p>{ingredient.ingredient}</p>
      </div> }
    </section>

  )
}

export default GameBoardCard;