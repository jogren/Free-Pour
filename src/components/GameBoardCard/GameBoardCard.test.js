import React from 'react';
import { shallow } from 'enzyme';
import GameBoardCard from './GameBoardCard';

describe('GameBoardCard', () => {
  let wrapper;
  const mockIngredient = { 
    measure: "2 tsp ", 
    ingredient: "Sugar", 
    guessed: false 
  }

  beforeEach(() => {
    wrapper = shallow(<GameBoardCard 
      index={0}
      ingredient={mockIngredient}
    />);
  })

  it('should take a snapshot when passed the correct data', () => {
    expect(wrapper).toMatchSnapshot();
  });
})