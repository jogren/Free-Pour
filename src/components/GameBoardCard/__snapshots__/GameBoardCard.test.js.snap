// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameBoardCard should take a snapshot when passed the correct data 1`] = `
<section>
  <div
    className="Game-card_div incorrect-guess"
  >
    <p>
      1
    </p>
  </div>
</section>
`;
